compclub/compclub/settings.py

```diff
@@ -19,11 +19,15 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ["SECRET_KEY"]
+env = os.environ.get('ENV', 'development').lower()
+if env == 'production':
+    # SECURITY WARNING: keep the secret key used in production secret!
+    SECRET_KEY = os.environ["SECRET_KEY"]
+    DEBUG = False
```

Comment:
Nice! But just keep in mind, you should not run the production web server with `python manage.py runserver`. You need to use a wsgi deployment (like gunicorn or meinheld)

https://docs.djangoproject.com/en/2.0/howto/deployment/wsgi/

Reply:
In that case, that would sound like a better idea. Let's try that. I'll look into setting this up in Docker.

Reply:
Feel free to take inspiration from https://github.com/csesoc/csesoc-website-v2/blob/master/Dockerfile